src/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfo.cs

```diff
@@ -251,12 +251,15 @@ internal void CopyRuntimeSettingsTo(JsonPropertyInfo other)
             
         internal abstract IList CreateConverterList();
 
+        internal abstract Type GetConcreteType(Type interfaceType);
```

Comment:
Why does this need to be abstract?

Reply:
It's abstract in the non-generic version so it can be called without using a generic. It is overridden in the generic implementations so it can leverage the generics in the implementation.